src/components/badges/gdbd.module.css

```diff
@@ -0,0 +1,19 @@
+.great-deal {
+  color: #149246;
+}
+
+.good-deal {
+  color: #81b040;
+}
+
+.fair-deal {
+  color: #eab000;
+}
+
+.high-deal {
+  color: #fa9000;
```

Comment:
We do have tailwind values but unfortunately, they have been diverted for high-deal and not-defined and are consumed in other places of the project. Will create some tech task to clean up and define some common color palette